index.html

```diff
@@ -380,7 +380,37 @@ <h5 class="modal-title">Other Settings</h5>
             }
         }
 
+        // Checks for any overrides from the querystring and sets the form values as appropriate
+        function checkOverrides() {
+            // Maps between the parameter name and the selector of the input element
+            const configMappings = {
+                "mem": "#inp_memsz",
+                "vgamem": "#inp_vmemsz",
+                "apic": "#inp_apic",
+                "acpi": "#inp_acpi",
+                "pci":  "#inp_pci",
+                "pcivga": "#inp_pcivga",
+                "floppy": "#inp_floppy",
+                "winnt": "#inp_winnt",
+            };
+            
+            const entries = Object.entries(configMappings);
+            for (const [parameter, id] of entries) {
+                var value = getParameterByName(parameter);
+                if (!value)
+                    continue;
+                var input = $(id);
+                // Explict check for checkbox.
+                if (input.prop('type') == "checkbox")
```

Comment:
I'd use the strict equality operator, `===`, here just for consistency, although it's not really necessary in this case.